src/Components/WebAssembly/Sdk/integrationtests/ServiceWorkerAssert.cs

```diff
@@ -6,7 +6,7 @@
 using System.IO;
 using System.Linq;
 using System.Text.Json;
-using Microsoft.AspNetCore.Razor.Tasks;
+using Microsoft.NET.Sdk.BlazorWebAssembly;
 
 namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
```

Comment:
I wanted to change these names, but it ends up being a lot more work to clean up the Razor SDK integration tests too. Leaving that for another day.